### DISCUSSION TITLE
Record/Map/Hash in Mantine Forms?

### DISCUSSION THREAD

**Discussion by indicozy** [mantinedev/mantine#4003] [Feature requests]
Hi there!

Is there a way to implement Record/Map/Hash in Mantine Forms?
For example (very rough):
```jsx
{
  userId: number;
  userFriends: {
  [key: number] : IAnotherUser
 }
}
```

**Comment by rtivital** [comment]
You can use any data types in form, for example Map:

```tsx
const form = useForm({ initialValues: { data: new Map() } })
```

**Comment by indicozy** [comment]
thanks!

ср, 5 апр. 2023 г. в 14:25, Vitaly Rtishchev ***@***.***>:
